modules/appeals_api/config/schemas/v2/200995_with_shared_refs.json

```diff
@@ -30,6 +30,7 @@
                 },
                 "claimantType": {"type": "string", "enum": [ "veteran", "spouse_of_veteran", "child_of_veteran", "parent_of_veteran", "other"] },
                 "claimantTypeOtherValue": { "type":  "string", "minLength": 1, "maxLength": 25 },
+                "alternateSigner": {"type": "string"},
```

Comment:
I think this can be removed since the signer details are in the headers now, right?

Reply:
removed